src/vdp_scanner.py

```diff
@@ -0,0 +1,296 @@
+"""Check current federal DotGov domains for a Vulnerability Disclosure Policy.
```

Comment:
Always good to define the acronym at the start.
```suggestion
"""Check current federal DotGov domains for a Vulnerability Disclosure Policy (VDP).
```